weasyl/test/test_collection.py

```diff
@@ -19,7 +19,22 @@ def offer(self):
         collection.offer(self.creator, self.s, self.collector)
 
     def count_collections(self, pending, rating=ratings.GENERAL.code):
-        return len(collection.select_manage(self.collector, rating, 10, pending))
+        return self.db.scalar(
```

Comment:
There was `collection.select_manage` =) It was only used for this set of tests, though, so I moved it here and took out the query string building. Note that it is still just a utility function and not a test.